docs/tutorial.md

```diff
@@ -18,6 +18,86 @@ The wizard will also ask if you'd like it setup any additional metadata files su
 > Ref: See [`zigmod init`](./commands/init.md) for more info.
 
 ---
+## Understanding `zig.mod`
+
+Running `zigmod init` will ask if You want to create an application or a
```

Comment:
can you link the inline code to https://github.com/nektro/zigmod/blob/master/docs/commands/init.md